### DISCUSSION TITLE
Learning Bootstrap

### DISCUSSION THREAD

**Discussion by jiresse218** [ColorlibHQ/AdminLTE#3544] [General]
Hello I am a beginner in bootstrap. I have learned some html, css and javascript. Where do i start from with bootstrap

**Comment by jiresse218** [comment]
Please help

**Reply by danny007in** [reply]
check this https://www.youtube.com/watch?v=QAgrHLtG1Yk&list=PL4cUxeGkcC9jE_cGvLLC60C_PeF_24pvv
next time as to bootstrap repo 😃
